### ISSUE TITLE
Carousel a11y issues

### ISSUE THREAD

**Issue by elenagarrone** [Legal-and-General/canopy#642]
The tab panel has an `aria-selected` attribute which is incorrect; it should be replaced with `aria-hidden` instead.
`aria-selected` should only go on the element that has `role="tab"` and should be working together with `aria-controls="<tabpanel-id>"`.

**Comment by elenagarrone** [comment]
There are other issues too e.g. `aria-labelledby`. 
See https://www.w3.org/TR/wai-aria-practices-1.1/examples/carousel/carousel-1.html for correct attributes.

**Comment by pauleustice** [comment]
Canopy no longer has a Carousel component so this is irrelevant
